clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/content/PendingIntentFactory.kt

```diff
@@ -69,8 +68,14 @@ internal object PendingIntentFactory {
             launchIntent.flags =
                 Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_SINGLE_TOP or Intent.FLAG_ACTIVITY_NEW_TASK
             var flagsLaunchPendingIntent = PendingIntent.FLAG_UPDATE_CURRENT
-            if (Build.VERSION.SDK_INT >= VERSION_CODES.S) {
-                flagsLaunchPendingIntent = flagsLaunchPendingIntent or PendingIntent.FLAG_IMMUTABLE
+            if (VERSION.SDK_INT >= VERSION_CODES.M) {
+                flagsLaunchPendingIntent = flagsLaunchPendingIntent or
+                        if (launchIntent.hasExtra(PTConstants.PT_INPUT_FEEDBACK)) {
+                            //  PendingIntents attached to actions with remote inputs must be mutable
+                            PendingIntent.FLAG_MUTABLE
```

Comment:
what about the warning on play console, how will we handle that

Reply:
That was related to implicit pending intents https://github.com/CleverTap/clevertap-android-sdk/issues/472
The launch intent above is explicit and hence shouldn't be a problem